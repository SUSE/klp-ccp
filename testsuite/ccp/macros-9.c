#define A
#define B
#define C

#define __str(a) #a
#define str(a) __str(|a|)

void pu_f(void)
{
	str(A B C);
}
