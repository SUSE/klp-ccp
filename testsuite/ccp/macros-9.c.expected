#define A

#define B

#define C

#define __str(a) #a

#define str(a) __str(|a|)

void klpp_pu_f(void)
{
	str(A B C);
}
